tests/scripts/run-throughput-perf.py

```diff
@@ -0,0 +1,362 @@
+# Main
+##########################################################################
+
+def main(args):
+    global dll_list
+
+    architecture, operating_system, build_type, run_type, clr_root, assembly_root, benchview_path = validate_args(args)
+    arch = architecture
+
+    current_dir = os.getcwd()
+    jit = "clrjit.dll"
+
+    # jit32 uses compatjit.dll
+    if architecture == 'x86jit32':
+        arch = 'x86'
+        jit = "compatjit.dll"
+
+    # Make sandbox
+    sandbox_path = os.path.join(clr_root, "sandbox")
+    if os.path.isdir(sandbox_path):
+        shutil.rmtree(sandbox_path, onerror=del_rw)
+
+    os.makedirs(sandbox_path)
+    os.chdir(sandbox_path)
+
+    # Set up paths
+    test_root = os.path.join(clr_root, 'bin', 'Tests', operating_system + '.' + arch + '.' + build_type)
+    core_root_path = os.path.join(test_root, 'Tests', 'Core_Root')
+
+    crossgen_path = os.path.join(core_root_path,'crossgen.exe')
+    jit_path = os.path.join(core_root_path, jit)
+
+    iterations = 5
+
+    # Run throughput testing
+    for dll_name in dll_list:
+        dll_file_name = dll_name + ".dll"
+        dll_path = os.path.join(assembly_root, dll_file_name)
+        dll_elapsed_times = runIterations(dll_file_name, dll_path, iterations, crossgen_path, jit_path, assembly_root)
+
+        if len(dll_elapsed_times) != 0:
+            if not benchview_path is None:
+                # Generate the csv file
+                csv_file_name = generateCSV(dll_name, dll_elapsed_times)
+
+                # For each benchmark, call measurement.py
+                measurement_args = ["py",
+                        os.path.join(benchview_path, "measurement.py"),
+                        "csv",
+                        os.path.join(os.getcwd(), csv_file_name),
+                        "--metric",
+                        "execution_time",
+                        "--unit",
+                        "milliseconds",
+                        "--better",
+                        "desc",
+                        "--append"]
+                log(" ".join(measurement_args))
+                proc = subprocess.Popen(measurement_args)
+                proc.communicate()
+            else:
+                # Write output to console if we are not publishing
+                log("%s" % (dll_name))
+                log("Duration: [%s]" % (", ".join(str(x) for x in dll_elapsed_times)))
+
+
+    # Upload the data
+    if not benchview_path is None:
+        # Call submission.py
+        submission_args = ["py",
+                os.path.join(benchview_path, "submission.py"),
+                "measurement.json",
+                "--build",
+                os.path.join(clr_root, "build.json"),
+                "--machine-data",
+                os.path.join(clr_root, "machinedata.json"),
+                "--metadata",
+                os.path.join(clr_root, "submission-metadata.json"),
+                "--group",
+                "CoreCLR",
```

Comment:
Do we want to have this uploaded as part of another group?  I am not a Benchview expert, but it looks like if we use the same group we will end up putting extra builds on the trendline for CoreCLR that will only have these Throughput tests.  @jorive @dpodder